scout/adapter/mongo/hgnc.py

```diff
@@ -347,15 +347,13 @@ def genes_by_alias(self, build=None, genes=None):
 
         for gene in genes:
             hgnc_id = gene["hgnc_id"]
-            hgnc_symbol = gene["hgnc_symbol"]
 
             for alias in gene["aliases"]:
                 if alias not in alias_genes:
                     alias_genes[alias] = {"true": None, "ids": set()}
 
                 alias_genes[alias]["ids"].add(hgnc_id)
-                if alias == hgnc_symbol:
```

Comment:
This might have worked before because gene symbols were up-to-date on the OMIM files?

Reply:
Yeah, and OMIM were doing a great job of it, but apparently just got a bit behind with the current release.